R/addins.R

```diff
@@ -33,21 +33,21 @@ devex_gadget <- function() {
         "stateRadio", "Project State", 
         choiceNames = list(
           shiny::span(
-            shiny::img(src = "https://assets.bcdevexchange.org/images/badges/inspiration.svg"), 
-            shiny::p("An idea being explored and shaped. Open for discussion, but may never go anywhere.")), 
+            shiny::img(src = "https://img.shields.io/badge/Lifecycle-Experimental-339999"), 
+            shiny::p("The project is in the very early stages of development. The codebase will be changing frequently.")), 
           shiny::span(
-            shiny::img(src = "https://assets.bcdevexchange.org/images/badges/exploration.svg"),
-            shiny::p("Being designed and built, but in the lab. May change, disappear, or be buggy.")),
+            shiny::img(src = "https://img.shields.io/badge/Lifecycle-Maturing-007EC6"),
+            shiny::p("The codebase is being roughed out, but finer details are likely to change.")),
           shiny::span(
-            shiny::img(src = "https://assets.bcdevexchange.org/images/badges/dormant.svg"), 
-            shiny::p("Not currently being worked on, but with plans to come back to the work in the future.")), 
+            shiny::img(src = "https://img.shields.io/badge/Lifecycle-Stable-97ca00"), 
+            shiny::p("The project is in a reliable state and major changes are unlikely to happen.")), 
           shiny::span(
-            shiny::img(src   = "https://assets.bcdevexchange.org/images/badges/delivery.svg"), 
-            shiny::p("In production, but maybe in Alpha or Beta. Intended to persist and be supported.")),
+            shiny::img(src   = "https://img.shields.io/badge/Lifecycle-Dormant-%23ff7f2a"), 
+            shiny::p("The project is currently not under active development, but there are plans to redevelop.")),
           shiny::span(
-            shiny::img(src = "https://assets.bcdevexchange.org/images/badges/retired.svg"), 
-            shiny::p("No longer being used or supported, recommend not using an alternative option."))), 
-        choiceValues = c("inspiration", "exploration", "dormant", "delivery", "retired")),
+            shiny::img(src = "https://img.shields.io/badge/Lifecycle-Retired-d45500"), 
+            shiny::p("The project is no longer being used and/or supported."))), 
+        choiceValues = c("experimental", "marturing", "dormant", "stable", "retired")),
```

Comment:
```suggestion
        choiceValues = c("experimental", "maturing", "stable", "dormant", "retired")),
```

typo, and the order was different than the selection list, so choosing "stable" inserted "dormant", and vice-versa